vcd/datasource_vcd_tm_region_zone.go

```diff
@@ -79,7 +79,7 @@ func resourceVcdTmRegionZoneRead(ctx context.Context, d *schema.ResourceData, me
 	return readDatasource(ctx, d, meta, c)
 }
 
-func setZoneData(d *schema.ResourceData, z *govcd.Zone) error {
+func setZoneData(_ *VCDClient, d *schema.ResourceData, z *govcd.Zone) error {
```

Comment:
Sorry, a bit confused. Why are we adding a blank VCDClient identifier which is not used afterwards?

Reply:
We're not in need it in this particular PR, but generic function signature requires it now. It is to be used in #1361 and possibly other more complicated entries